gpustack/config/envs.py

```diff
@@ -7,6 +7,7 @@
 DB_POOL_SIZE = int(os.getenv("GPUSTACK_DB_POOL_SIZE", 5))
 DB_MAX_OVERFLOW = int(os.getenv("GPUSTACK_DB_MAX_OVERFLOW", 10))
 DB_POOL_TIMEOUT = int(os.getenv("GPUSTACK_DB_POOL_TIMEOUT", 30))
+DB_PORT = int(os.getenv("POSTGRES_PORT", 5432))
```

Comment:
Do we need to keep the naming style consistent?
Should we add it to start parameter given that it affects installation?